saleor/dashboard/order/forms.py

```diff
@@ -248,18 +247,10 @@ def save(self, commit=True):
 class OrderNoteForm(forms.ModelForm):
     class Meta:
         model = OrderNote
-        fields = ['content', 'is_public']
+        fields = ['content']
```

Comment:
Do we still want to use a one to many relation for the dashboard note form?

Reply:
Do You mean to leave only one-time comment here as well?

Reply:
This one should use a relation - admins are able to add multiple notes to an order.